libs/designsystem/src/lib/components/modal/services/alert.helper.ts

```diff
@@ -60,11 +56,6 @@ export class AlertHelper {
   }
 
   private getCancelBtnText(config: AlertConfig) {
```

Comment:
Consider remvoing this line competely and just use `config.cancelBtn` on line 34 instead.

Reply:
Fixed in a1a30352